tests/rados/rados_config_parameters.ini

```diff
@@ -0,0 +1,38 @@
+#MSG version 2 parameters
```

Comment:
test configs needs to be moved under : ` conf/pacific/rados/test-confs ` or `conf/quincy/rados/test-confs ` directories

Reply:
Moved the rados_config_parameters.ini file under conf/quincy/rados/test-confs directory.